jupyter-spark/extensions/spark.js

```diff
@@ -1,4 +1,5 @@
-var API = "/spark/api/v1";
+var BASE_URL = Jupyter.utils.get_body_data("baseUrl") || "/";
```

Comment:
I don't know why (WHY?!1?) jQuery is doing that, but it seems as if access to data parameters [are stored camel cased by default](https://github.com/jquery/jquery/blob/58c6ca9822afa42d3b40cca8edb0abe90a2bcb34/src/data/Data.js#L50-L75). There is also lots [use of the camel-cased name](https://github.com/jupyter/notebook/search?utf8=%E2%9C%93&q=get_body_data) in the Notebook source, so this should be fine™.